tests/integration/conftest.py

```diff
@@ -171,7 +172,8 @@ def print_dialog(mocker, homedir):
         )
         controller.authenticated_user = factory.User()
         controller.qubes = False
-        export_device = conversation.StubExportDevice(controller)
+        export_device = conversation.ExportDevice(controller)
+        export_device._export_service = mocker.MagicMock(spec=Export)
```

Comment:
> From there testing the dialog could be done with a "test" device. In practice, with a device which state would be controlled by the test setup, and which (non-GUI) export_service would be mocked.

This works exactly as I expected when writing the quotes comment. :point_up: 

----

Interestingly (though that's out of scope for this PR), our integration tests don't really test the export process beyond the initial rendering of the dialogs. If the Qubes OS-specific behavior is not mocked, cancelling the _dom0_ dialogs still results in successful tests. I don't think that's necessarily _wrong_, but there may be an opportunity here for more exhaustive testing?